todoTask/todoTask.js

```diff
@@ -1,85 +1,59 @@
-﻿// ------------------------global varibals--------------------
-let globalID = 0;
-// -------------------------functions---------------------------
-const getId = () => (globalID += 1);
+const getId = (() => {
+    let globalID = 0;
+    return () => (globalID += 1);
+})();
 
-const makeTodo = (title, complete) => {
-    return {
-        id: getId(),
-        title,
-        complete,
-    };
-};
+const createTodo = (title, complete = false) => ({
+    id: getId(),
+    title,
+    complete,
+});
 
-const makeTodos = (n) => {
-    let todos = [];
-    for (let i = 1; i <= n; i++) {
```

Comment:
Start `for` always from `0` (except if it's required by algorithm).
And then `i < n`